tests/test_etherscan.py

```diff
@@ -0,0 +1,18 @@
+import json
+
+import pytest
+from requests import Response
+
+
+@pytest.fixture
+def etherscan_abi_response(mocker):
+    response = mocker.MagicMock(spec=Response)
+    f = open("api.json")
+    response.json.return_value = json.load(f)
+    f.close()
+    return response
+
+
+def test(mocker):
+    get_patch = mocker.patch("ape_etherscan.explorer.requests")
+    get_patch.get.return_value = etherscan_abi_response
```

Comment:
Right now, this test is setting a method as the return_value rather than the actual fixture.

To fix, have the test accept the `etherscan_abi_response` (it is automatically passed since it is a fixture).

Reply:
so `assert etherscan_abi_response`?

Reply:
Like this 

```
test_get_contract_type(mocker, etherscan_abi_response)
```

That way, it uses the fixture instead of the method when setting up the mock.